src/components/SelectionList/types.ts

```diff
@@ -674,6 +674,8 @@ type SelectionListProps<TItem extends ListItem> = Partial<ChildrenProps> & {
 
     /** Whether to add bottom safe area padding to the content. */
     addOfflineIndicatorBottomSafeAreaPadding?: boolean;
+
+    fixedNumItems?: number;
```

Comment:
```suggestion
    fixedNumItemsForLoader?: number;
```
`fixedNumItems` won't be clear that it is for the loader in the SelectionList. Let's update the name of the prop here.